src/core/transports/transport.ts

```diff
@@ -46,7 +47,8 @@ export default class Transport {
     name: string,
     priority: number,
     key: string,
-    options: any
+    options: any // TODO
```

Comment:
can't we track this somewhere else?

Reply:
(the TODO)

Reply:
I'll just remove it, I'm not going to do this as part of this PR now. I'm conscious of trying to make these changes incrementally